### PR TITLE
Added execute transfer functionality

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

File: executors/ethereum/migrationBatchExecutor.go [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+func (executor *migrationBatchExecutor) checkRelayersSigsAndQuorum(relayers []common.Address, quorum *big.Int) ([][]byte, error) {
+	sameMessageHashSignatures := executor.getSameMessageHashSignatures()
+	validSignatures := executor.getValidSignatures(sameMessageHashSignatures)
+	return executor.checkQuorum(relayers, quorum, validSignatures)
+}
+
+func (executor *migrationBatchExecutor) getSameMessageHashSignatures() []SignatureInfo {
+	filtered := make([]SignatureInfo, 0, len(executor.signatures))
+	for _, sigInfo := range executor.signatures {
+		if sigInfo.MessageHash != executor.batch.MessageHash.String() {
```
Comment (dragos-rebegea) [inline executors/ethereum/migrationBatchExecutor.go:176]:
```suggestion
        expectedSignature := executor.batch.MessageHash.String()
	for _, sigInfo := range executor.signatures {
		if sigInfo.MessageHash != expectedSignature {
```

File: executors/ethereum/migrationBatchExecutor.go [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+		if found {
+			executor.logger.Warn("found a multiple relayer sig info, ignoring",
+				"address", sigInfo.Address)
+			continue
+		}
+
+		whitelistedRelayers[relayerAddress] = sigInfo
+	}
+
+	result := make([][]byte, 0, len(whitelistedRelayers))
```
Comment (dragos-rebegea) [inline executors/ethereum/migrationBatchExecutor.go:266]:
```suggestion
	validSignatures := make([][]byte, 0, len(whitelistedRelayers))
```

File: executors/ethereum/migrationBatchExecutor.go (hunk 1 above)
Comment (iulianpascalau) [reply executors/ethereum/migrationBatchExecutor.go:176]: done

File: executors/ethereum/migrationBatchExecutor.go (hunk 2 above)
Comment (iulianpascalau) [reply executors/ethereum/migrationBatchExecutor.go:266]: renamed

**Review by sstanculeanu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**